src/components/draft-js-editor/DraftJSEditor.js

```diff
@@ -111,7 +111,7 @@ class DraftJSEditor extends React.Component<Props> {
                 ariaAutoComplete: inputProps['aria-autocomplete'],
                 ariaExpanded: inputProps['aria-expanded'],
                 ariaOwneeID: inputProps['aria-owns'],
-                role: inputProps.role,
+                role: inputProps.textbox,
```

Comment:
This change will set the `role` to the value of the `inputProps.textbox` prop, which appears to be undefined for most usages. Can we instead update the calling components to pass in the proper `role` prop?

Reply:
Done!

Reply:
@msorianoBox I think what is confusing here to me is that we have `<DraftJSEditor>` (component) which has a prop called `inputProps.role` but we're now ignoring that prop and using `inputProps.textbox`? So the calling components will go from
```
<DraftJSEditor inputProps={{role: 'textbox'}} />
```
to
```
<DraftJSEditor inputProps={{textbox: 'textbox'}} />
```

Is this expected? cc: @jstoffan

Reply:
If you want to ignore `inputProps.role` and **always** force it to `textbox` then wouldn't this work:

```
...
role: 'textbox',
```